content/influxdb/cloud-dedicated/get-started/query.md

```diff
@@ -185,10 +191,62 @@ WHERE
 
 {{< tabs-wrapper >}}
 {{% tabs %}}
+[influx3 CLI](#influx3-cli)
 [Python](#)
 [Go](#)
 {{% /tabs %}}
+{{% tab-content %}}
+<!--------------------------- BEGIN influx3 CONTENT --------------------------->
+
+Query InfluxDB v3 using SQL and the `influx3` CLI, part of the [`InfluxCommunity/pyinflux3`](https://github.com/InfluxCommunity/pyinflux3) community repository.
+The following steps include setting up a Python virtual environment already
+covered in [Get started writing data](/influxdb/cloud-dedicated/get-started/write/?t=Python#write-line-protocol-to-influxdb).
+_If your project's virtual environment is already running, skip to step 3._
+
+1.  Setup your Python virtual environment.
+    Inside of your project directory:
+
+    ```sh
+    python -m venv envs/virtual-env
+    ```
+
+2. Activate the virtual environment.
+
+    ```sh
+    source ./envs/virtual-env/bin/activate
+    ```
 
+3. Install the following dependencies:
+
+    {{< req type="key" text="Already installed in the [Write data section](/influxdb/cloud-dedicated/get-started/write/?t=Python#write-line-protocol-to-influxdb)" color="magenta" >}}
+
+    - `pyarrow` {{< req text="\*" color="magenta" >}}
+    - `flightsql-dbapi` {{< req text="\*" color="magenta" >}}
+    - `pyinflux3` {{< req text="\*" color="magenta" >}}
+
+4. Create the `config.json` configuration.
+
+    ```sh
+    influx3 config --name="my-config" --database="DATABASE_NAME" --host="cluster-id.influxdb.io" --token="DATABASE_TOKEN" --org="INFLUX_ORG_ID"
+    ```
```

Comment:
- I generally try to prevent codeblock overflow and scrolling where possible. Feel free to ignore.
- Is org ID required here? I know it isn't needed, but is the client requiring it?

~~~suggestion
    ```sh
    influx3 config \
      --name="my-config" \
      --database="DATABASE_NAME" \
      --host="cluster-id.influxdb.io" \
      --token="DATABASE_TOKEN" \
      --org="INFLUX_ORG_ID"
    ```
~~~

Reply:
> * Is org ID required here? I know it isn't needed, but is the client requiring it?

Yes, unfortunately, the client still requires `--org`.